js/templates/template-page-settings.js

```diff
@@ -379,10 +379,13 @@ Updated
             <hr>
              <div>
                 <div class='pagedetails__header'>
-                    <h3 class='page_title'>Weather</h3>
```

Comment:
indentation's a bit all over the place here; switch it all out for tabs, would you?